common/src/main/java/tachyon/exception/AbstractTachyonException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.exception;
+
+/**
+ * Abstract class for Tachyon exceptions.
+ *
+ */
+public abstract class AbstractTachyonException extends Exception {
+  private static final long serialVersionUID = -5227424702682358256L;
```

Comment:
Hmm looking at the answer it seems like its not necessary for an abstract class (since it will never be instantiated and need to be serialized and the classes inheriting from it should have their own UID)?